src/pages/faq/faq.mdx

```diff
@@ -0,0 +1,133 @@
+---
+name: FAQ
+route: /faq
+---
+
+# FAQ
+
+## How can I listen to form changes in the standalone component?
+
+When using JSON Forms within your react app, you probably want to access the current form data. 
+For this purpose you can use the `onChange` prop, which returns the form data and possible validation errors provided by AJV.
+To get you started, we built two simple examples for you. 
+
+The first example showcases a class component:
+```
+import React from 'react';
+import { JsonForms } from '@jsonforms/react';
+import { person } from '@jsonforms/examples';
+import { materialRenderers } from '@jsonforms/material-renderers';
+
+const schema = person.schema;
+const uischema = person.uischema;
+const data = person.data;
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data
+    };
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <JsonForms
+          schema={schema}
+          uischema={uischema}
+          data={data}
+          renderers={materialRenderers}
+          onChange={({ errors, data }) => this.setState({data})}
+        />
+      </div>
+    );
+  }
+}
+
+export default App;
+```
+
+The second example uses the functional component approach. Therefore we use the `useState` hook, which provides the data (first parameter) and a function to update the data (second parameter).
+```
+import React, { useState } from 'react';
+import { JsonForms } from '@jsonforms/react';
+import { person } from '@jsonforms/examples';
+import { materialRenderers } from '@jsonforms/material-renderers';
+
+const schema = person.schema;
+const uischema = person.uischema;
+const data = person.data;
+
+function App() {
+  const [stateData, setData] = useState(data);
+
+  return (
+    <div className="App">
+      <JsonForms
+        schema={schema}
+        uischema={uischema}
+        data={stateData}
+        renderers={materialRenderers}
+        onChange={({ errors, data }) => setData(data)}
+      />
+    </div>
+  );
+}
+
+export default App;
+```
+
+
+## How can I integrate a custom renderer?
+
+If you want to change the way a field is rendered (for example showing stars for a rating instead of a number), you can create your own renderer. 
```

Comment:
Providing a custom renderer gives you full control of the rendered output, enabling you to integrate any custom behavior or pre-built component you'd like.